core/src/sim.c

```diff
@@ -247,9 +231,10 @@ int handle_potential_stopping_condition(Simmer *simmer) {
   SimmedPlay *tentative_winner = simmer->simmed_plays[0];
   double mu = tentative_winner->win_pct_stat->mean;
   double stderr = get_standard_error(tentative_winner->win_pct_stat, zval);
-  int new_ignored = 0;
+  int total_ignored = 0;
   for (int i = 1; i < simmer->num_simmed_plays; i++) {
```

Comment:
if we're counting all ignored plays shouldn't this start at `i = 0` ?

Reply:
I thought that it would be impossible for the first play to ever be ignored and that we were only concerned with comparing every other play against the tentative winner. Let me know what I'm missing here.

Reply:
you might be right here.